src/components/Events.vue

```diff
@@ -57,5 +68,6 @@
     text-decoration: underline;
     font-family: $font-raleway;
     font-size: 18px !important;
```

Comment:
How come this has been set to 18px? Also we have variables for font-sizes.

Reply:
@suzannahmahala I'm not sure what the font size is. I can add it if you tell me what it is, or we can address when coming to refine the Events component. Non of the components are final in this branch. I've just been addressing the base styling for mobile.